src/main/java/org/team1540/robot2024/commands/indexer/StageTrampCommand.java

```diff
@@ -14,12 +14,10 @@ public StageTrampCommand(Tramp tramp, Indexer indexer) {
         this.indexer = indexer;
         addRequirements(tramp, indexer);
     }
-
     @Override
     public void initialize() {
         tramp.setPercent(0.5);
         indexer.setFeederVelocity(-600);
-        indexer.setIntakePercent(0.5);
```

Comment:
Isn't this needed to actually push the note into the feeder to direct?

Reply:
Simon had said that the feeder will be touching the note while it is in the indexer. Would that not be enough to push it into the tramp?

Reply:
If the note is being contacted by the intake belts (which it is here), having the intake not spin will actively oppose the motion that we want the note to take.